src/main/java/de/tudarmstadt/ukp/jwktl/api/berkeley/BerkeleyConfigurationModel.java

```diff
@@ -0,0 +1,254 @@
+ * Copyright 2013
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+package de.tudarmstadt.ukp.jwktl.api.berkeley;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.sleepycat.compat.DbCompat;
+import com.sleepycat.persist.model.AnnotationModel;
+import com.sleepycat.persist.model.ClassMetadata;
+import com.sleepycat.persist.model.EntityMetadata;
+import com.sleepycat.persist.model.FieldMetadata;
+import com.sleepycat.persist.model.KeyField;
+import com.sleepycat.persist.model.NotPersistent;
+import com.sleepycat.persist.model.NotTransient;
+import com.sleepycat.persist.model.PrimaryKey;
+import com.sleepycat.persist.model.PrimaryKeyMetadata;
+import com.sleepycat.persist.model.Relationship;
+import com.sleepycat.persist.model.SecondaryKey;
+import com.sleepycat.persist.model.SecondaryKeyMetadata;
+
+import de.tudarmstadt.ukp.jwktl.api.entry.*;
+
+public class BerkeleyConfigurationModel extends AnnotationModel {
+	private static class EntityInfo {
+		PrimaryKeyMetadata priKey;
+		Map<String, SecondaryKeyMetadata> secKeys = new HashMap<String, SecondaryKeyMetadata>();
+	}
+
+	private Map<String, ClassMetadata> classMap;
+	private Map<String, EntityInfo> entityMap;
+
+	/**
+	 * Constructs a model for annotated entity classes.
+	 */
+	public BerkeleyConfigurationModel() {
+		super();
+		classMap = new HashMap<String, ClassMetadata>();
+		entityMap = new HashMap<String, EntityInfo>();
+
+		addPersistentMetadata(Pronunciation.class.getName());
+		addPersistentMetadata(Quotation.class.getName());
+		addPersistentMetadata(WikiString.class.getName());
+		addPersistentMetadata(WiktionaryEntry.class.getName());
+		addPersistentMetadata(WiktionaryExample.class.getName());
+		addPersistentMetadata(WiktionaryRelation.class.getName());
+		addPersistentMetadata(WiktionarySense.class.getName());
+		addPersistentMetadata(WiktionaryTranslation.class.getName());
+		addPersistentMetadata(WiktionaryWordForm.class.getName());
+
+		try {
+			addClassMetadata(WiktionaryPage.class.getName());
+		} catch (Exception e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();

```

Comment:
why eat this exception instead of throwing? We should also check the return value of addClassMetadata (could be null for ClassNotFoundException)